apis/v1alpha1/shared_types.go

```diff
@@ -181,23 +194,25 @@ type RouteGatewayStatus struct {
 	//
 	// +listType=map
 	// +listMapKey=type
+	// +kubebuilder:validation:MinItems=1
```

Comment:
Same comment here - I don't think we should tighten validation as part of this release. It does seem like we're compiling an increasing number of changes for v1alpha2 though.

Reply:
@robscott done.